test/util/server/server.go

```diff
@@ -137,6 +137,11 @@ func DefaultMasterOptionsWithTweaks(startEtcd, useDefaultPort bool) (*configapi.
 	}
 
 	masterConfig.ImagePolicyConfig.ScheduledImageImportMinimumIntervalSeconds = 1
+	allowedRegistries := append(
+		*configapi.DefaultAllowedRegistriesForImport,
+		configapi.RegistryLocation{DomainName: "127.0.0.1:*"},
```

Comment:
Hrm, any reason not to allow localhost always?  Trying to think of a security reason